load-test-framework/src/main/java/com/google/pubsub/flic/common/LatencyDistribution.java

```diff
@@ -146,4 +146,26 @@ public void recordLatency(long latencyMs) {
       bucketValues[LATENCY_BUCKETS.length - 1]++;
     }
   }
+
+  public void recordLatency(long latencyMs, int addCount) {
```

Comment:
May want to make this recordLatencyBatch, and have a recordLatency that just does 1, since we use that a lot and it may look nicer.  Also maybe batchSize instead of addCount. We prefer not to use overloading whenever possible.